packages/block-library/src/navigation/edit/index.js

```diff
@@ -794,7 +798,18 @@ function Navigation( {
 						{ hasResolvedCanUserDeleteNavigationMenu &&
 							canUserDeleteNavigationMenu && (
 								<NavigationMenuDeleteControl
-									onDelete={ resetToEmptyBlock }
+									onDelete={ ( deletedMenuTitle = '' ) => {
```

Comment:
Nitpick: can't we just get the title from the edit component instead of having the `NavigationMenuDeleteControl` forced to "know" the title?

Reply:
I guess we could. I suppose it's just that

```js
const [ title, updateTitle ] = useEntityProp(
		'postType',
		'wp_navigation',
		'title'
	);
```

...requires that we're within an `EntityProvider`. `edit.js` isn't within that context.